### DISCUSSION TITLE
help with my domain

### DISCUSSION THREAD

**Discussion by rxvan** [vercel/vercel#6546] [Help] [answered]
so basically, i bought a domain to use for vercel, but i realised that vercel was not capable of doing what i wished to do, so i bought a vps;
I have linked the domain to my vps now, yet it still redirects to Vercel and i keep getting errors when using my domain.
I dont know what to do and its honestly frustrating since my VPS doesn't have a lot of time on it.
Any help is appreciated.

edit:
not sure if it helps but:
My domain: https://rxvan.wtf
The error im getting:  [click](https://cdn.upload.systems/uploads/zfqm61lX.png)

i know its going to vercel because the icon also changes from blank to vercels.

**Comment by leerob** [comment, marked as answer]
Could you contact Vercel support, please?

**Reply by rxvan** [reply]
i have, but i havent gotten a reply yet. im not sure what the issue is and its honestly frustrating.

**Reply by leerob** [reply]
Could you let me know the ticket number? I will let the team know! I'm sorry about this.
